sema4ai/vscode-client/src/robo/agentPackage.ts

```diff
@@ -134,7 +135,7 @@ export const createAgentPackage = async (): Promise<void> => {
         const errorMsg = "Error creating Agent Package at: " + targetDir;
         logError(errorMsg, err, "ERR_CREATE_ACTION_PACKAGE");
         OUTPUT_CHANNEL.appendLine(errorMsg);
```

Comment:
If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).